src/treeview/tree-node.types.ts

```diff
@@ -7,6 +7,8 @@ export interface Node {
 	id?: string;
 	active?: boolean;
 	disabled?: boolean;
+	// selectable is only valid for nodes with childre. If true, the node won't become selected and if clicked it would only expand/compress itself
```

Comment:
```suggestion
	// selectable is only valid for nodes with children. If true, the node won't become selected and if clicked it would only expand/compress itself
```